src/3d/qgs3daxis.h

```diff
@@ -34,11 +34,11 @@
 #include <Qt3DRender/QGeometryRenderer>
 
 #include <QtWidgets/QMenu>
+#include "qgs3dmapsettings.h"
```

Comment:
it seems like #include of qgs3daxissettings.h should be enough? (to limit the amount of recursive includes that slow down compilation)

Reply:
which one should I remove?